test/boulder-tools/build.sh

```diff
@@ -38,9 +38,10 @@ wait
 
 # grpc uses a version attestation variable of the form grpc.SupportPackageIsVersionN
 # where N is the minor release version of the protoc-gen-go version used to
```

Comment:
Is this comment about the minor release version correct? AFAICT they don't have numbered releases, but I could be wrong.